src/index.ts

```diff
@@ -1 +1,107 @@
+/**
+ * > Template to kickstart creating a Node.js module using TypeScript and VSCode
+ *
+ * Inspired by [node-module-boilerplate](https://github.com/sindresorhus/node-module-boilerplate)
+ *
+ * ## Features
+ *
+ * - [Semantic Release](https://github.com/semantic-release/semantic-release)
+ * - [Issue Templates](https://github.com/ryansonshine/typescript-npm-package-template/tree/main/.github/ISSUE_TEMPLATE)
+ * - [GitHub Actions](https://github.com/ryansonshine/typescript-npm-package-template/tree/main/.github/workflows)
+ * - [Codecov](https://about.codecov.io/)
+ * - [VSCode Launch Configurations](https://github.com/ryansonshine/typescript-npm-package-template/blob/main/.vscode/launch.json)
+ * - [TypeScript](https://www.typescriptlang.org/)
+ * - [Husky](https://github.com/typicode/husky)
+ * - [Lint Staged](https://github.com/okonet/lint-staged)
+ * - [Commitizen](https://github.com/search?q=commitizen)
+ * - [Jest](https://jestjs.io/)
+ * - [ESLint](https://eslint.org/)
+ * - [Prettier](https://prettier.io/)
+ * - [API Documentation Generation](https://stackoverflow.com/a/57052392/955091)
+ *   - [TypeDoc](https://typedoc.org/)
+ *   - [typedoc-plugin-markdown](https://github.com/tgreyuk/typedoc-plugin-markdown/tree/master/packages/typedoc-plugin-markdown)
+ *   - [concat-md](https://github.com/ozum/concat-md)
+ *
+ * ## Getting started
+ *
+ * ### Set up your repository
+ *
+ * **Click the "Use this template" button.**
+ *
+ * Alternatively, create a new directory and then run:
+ *
+ * ```bash
+ * curl -fsSL https://github.com/ryansonshine/typescript-npm-package-template/archive/main.tar.gz | tar -xz --strip-components=1
+ * ```
+ *
+ * Replace `FULL_NAME`, `GITHUB_USER`, and `REPO_NAME` in the script below with your own details to personalize your new package:
+ *
+ * ```bash
+ * FULL_NAME="John Smith"
+ * GITHUB_USER="johnsmith"
+ * REPO_NAME="my-cool-package"
+ * sed -i.mybak "s/\([\/\"]\)(ryansonshine)/$GITHUB_USER/g; s/typescript-npm-package-template\|my-package-name/$REPO_NAME/g; s/Ryan Sonshine/$FULL_NAME/g" package.json package-lock.json README.md
```

Comment:
I have tested https://github.com/Atry/typescript-npm-package-template/tree/main and found that `src/index.ts` needs to be fixed as well, and https://github.com/ryansonshine/typescript-npm-package-template/issues/35#issuecomment-2044217656 is also needed.

```suggestion
 * sed -i.mybak "s/\([^@]\)ryansonshine/\1$GITHUB_USER/g; s/typescript-npm-package-template\|my-package-name/$REPO_NAME/g; s/Ryan Sonshine/$FULL_NAME/g" package.json package-lock.json README.md src/index.ts
```